pkgs/top-level/ocaml-packages.nix

```diff
@@ -291,6 +295,10 @@ let
 
     gmap = callPackage ../development/ocaml-modules/gmap { };
 
+    gnuplot = callPackage ../development/ocaml-modules/gnuplot {
+      inherit (pkgs) gnuplot;
```

Comment:
Please similarly fix the input to `metrics-unix`. Thanks.

Reply:
There's no `metrics-unix` on this branch; did you mean to comment somewhere else?

Reply:
There is one in `master`. Please rebase and we’ll be good to go.

Reply:
Rebase done.

Reply:
Cool. Now there is a `metrics-unix` whose input you can fix.

Reply:
I don't understand what you're asking.

Also, what does `metrics-unix` have to do with this package?  Shouldn't that be a separate issue/PR?

Reply:
The `ocamlPackages.metrics-unix` package takes as input `gnuplot` from the main package set. This PR changes this input: it gives `ocamlPackages.gnuplot` instead of `pkgs.gnuplot`.

Reply:
Sorry, didn't see that.  Done, metrics-unix builds correctly.